src/components/album/AlbumChat.js

```diff
@@ -1,23 +1,32 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import io from 'socket.io-client';
 
 const AlbumChat = ({ apiUrl }) => {
-  useEffect(() => {
+  const [message, setMessage] = useState('');
+
+  // Nandle sending a chat message
+  const handleSendMessage = () => {
     const socket = io(process.env.REACT_APP_SOCKET_BASE_PATH);
+    const regexPattern = /\/albums\/([a-zA-Z0-9]+)/;
+    const match = apiUrl.match(regexPattern);
```

Comment:
Instead of having to do this here and in `useEffect` we can just create a variable (doesn't need to be component state, just a regular `const` is fine here) at the top of the component that calculates the albumId or sets it to `null` or undefined otherwise.